### ISSUE TITLE
Size of the objects

### ISSUE THREAD

**Issue by chenyingjie** [CryoVR/CryoVR_GridClipping#17]
The workstation is super big in my goggle. Did you purposely enlarge it?

**Comment by Wandayu** [comment]
Yes, we enlarged it so that the user can see the details inside and thus better understand the whole process.
